src/sentry/static/sentry/app/views/organizationIntegrations/installedPlugin.tsx

```diff
@@ -58,6 +60,20 @@ export class InstalledPlugin extends React.Component<Props> {
     );
   };
 
+  updatePluginEnableStatus = async (enabled: boolean) => {
+    const {organization, projectItem, plugin} = this.props;
+    if (enabled) {
+      await this.pluginUpdate({enabled});
+    } else {
+      await this.props.api.requestPromise(
+        `/projects/${organization.slug}/${projectItem.projectSlug}/plugins/${plugin.id}/`,
+        {
+          method: 'DELETE',
```

Comment:
@NisanthanNanthakumar I am confused why in the `else` condition, you are actually deleting the installation. I would think all you would need to do here is just: `await this.pluginUpdate({enabled: false});`. Does that not work?

Reply:
@scefali I tried that and it did not work. We use the DELETE call in the disable button in the `legacy integration detailed view`.

Reply:
@NisanthanNanthakumar got it. Let's update `pluginUpdate` to take an optional argument for the `method` and default it to `POST` so we can use that function here instead of having to use `this.props.api.requestPromise` here.